notebooks/Repo_Analysis.ipynb

```diff
@@ -0,0 +1,1673 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    !python -m srcopsmetrics.cli -clr $repo_slug -e Issue,PullRequest
</pre><p>If we are planning to use the MI tool from within the notebook, do we also want to save a the raw Issue and PullRequest data on S3? I believe <code>-clr</code> only gets the data locally.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-services-group/metrics/pull/20/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
I think the metrics table is be a transformed table, mostly a subset of the original github data, it seems to omit some columns from the raw data. One possible benefit of storing the raw data could be to facilitate future re-analysis (eg ability to add additional metrics on the data belonging to the same time frame) and reproducibility. If we think the metrics data stores all original columns that will be important to us, there is no need of the raw data.

Reply:
Oh yeah I agree, the metrics data should include all the columns we are interested in. If there's additional columns in future we can add it to the same table with a one time data fetch for the past data, wdyt?

Reply:
sounds good!
/lgtm